src/Widgets/CanvasItem.vala

```diff
@@ -76,6 +83,14 @@ public class GtkCanvas.CanvasItem : Clutter.Actor {
     }
     private double _rotation = 0.0;
 
+    /**
+    * Fill color of the shape
+    */
+    public string color;
```

Comment:
I think this doesn't belong on the "Abstract" class that all items will use. Each shape might have it's own implementation. And it should also be a a property implementing the get/set methods :)

Reply:
Each shape should implement their own `background_color: Clutter.Color.from_string (color)` like so :)